### ISSUE TITLE
[Next] defineI18nRoute not register the route if ref(false) below it

### ISSUE THREAD

**Issue by Applelo** [nuxt-modules/i18n#1651]
### Version
@nuxtjs/i18n: 8.0.0-beta.3
nuxt: 3.0.0-rc.13

### @nuxtjs/i18n configuration

```
i18n: {
    defaultLocale: 'fr_FR',
    locales: [{ code: 'fr_FR', iso: 'fr-FR', file: 'fr.yml' }],
    langDir: './locales/',
  }
```

### Reproduction Link

[Reproduction link](https://stackblitz.com/edit/nuxt-i18n-nested-routes-gf729y?file=pages/login.vue)

### Steps to reproduce

Create a file using `defineI18nRoute` method and put under `false`

### What is Expected?

To be registered by `nuxt/i18n`

### What is actually happening?

The route is not registered by `nuxt/i18n`

**Comment by Applelo** [comment]
Ok, happen an another time in my current project with just a reference of `false` in my `<template></template>`

**Comment by Applelo** [comment]
Ok it seems like to be fix with 8.0.0-beta.4 :) I close the issue
